tests/system/python/packages/test_lab.py

```diff
@@ -0,0 +1,555 @@
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Lab packages System tests
+"""
+
+__author__ = "Yash Tatkondawar"
+__copyright__ = "Copyright (c) 2019 Dianomic Systems Inc."
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+import subprocess
+import http.client
+import json
+import pytest
+import os
+import time
+import urllib.parse
+
+
+# TODO:pass version to install script
+def setup_module(module):
+    try:
+        subprocess.run(["cd $FOGLAMP_ROOT/tests/system/lab && ./install"], shell=True, check=True)
```

Comment:
May be we should also put `subprocess.run(["cd $FOGLAMP_ROOT/tests/system/lab && ./remove"], shell=True, check=True)` here before install. to purge / clean everything before we start. and add a flag to run cleanup (defaults to yes) to run remove at the time of teardown.  these improvements needs to be considered when we set this up with CI.